platform/storage/zephyr/nvs/src/mender-storage.c

```diff
@@ -268,8 +267,10 @@ mender_storage_delete_device_config(void) {
     return MENDER_OK;
 }
 
-mender_err_t
-mender_storage_set_provides(mender_key_value_list_t *provides) {
+#ifdef CONFIG_MENDER_FULL_PARSE_ARTIFACT
+#ifdef CONFIG_MENDER_PROVIDES_DEPENDS
+static mender_err_t
+mender_set_provides(mender_key_value_list_t *provides, const int filename) {
```

Comment:
A bit strange to call the argument `filename` is this Zephyr-specific code where it is not a file name.